steampy/client.py

```diff
@@ -35,10 +36,28 @@ def __init__(self, api_key: str) -> None:
         self.market = SteamMarket(self._session)
         self.chat = SteamChat(self._session)
 
-    def login(self, username: str, password: str, steam_guard: str) -> None:
-        self.steam_guard = guard.load_steam_guard(steam_guard)
+    def login(self, username: str, password: str, third, saveCallback: Callable = None, steamAuth: dict = {}) -> None:
+
         self.username = username
-        LoginExecutor(username, password, self.steam_guard['shared_secret'], self._session).login()
+
+        if type(third) is str:
+            self.login_mobile(password, third)
+        elif callable(third):
+            self.login_email_auth(password, third, saveCallback, steamAuth)
+        else:
+            raise TypeError("Invalid third parameter type")
+
+
+    def login_email_auth(self, password: str, authCallback: Callable, saveCallback: Callable, steamAuth: dict):
```

Comment:
Email login can't really be automated on the FIRST login since it would require you to have code to access the person's email. Although you can save the cookies and use them in later logins, like the browser does, this means that you only have to input the steam guard code once.

IMHO this is my maximum automation the email login can reach before getting side tracked from the main focus of this library. Having some experience with other steambots I really do think that the first login via email requiring the user to go to the email is not much of deal since that file can be shared across mutliple machines.